files/build_templates/syncd.service.j2

```diff
@@ -1,30 +1,30 @@
 [Unit]
 Description=syncd service
 Requires=database.service updategraph.service
-{% if sonic_asic_platform == 'broadcom' %}
+Before=ntp-config.service
+After=database.service updategraph.service swss.service interfaces-config.service
+{%- if sonic_asic_platform == 'mellanox' %}
+Wants=mlnx-asic-syncd.service
+After=mlnx-asic-syncd.service
+{%- endif %}
+{%- if sonic_asic_platform == 'broadcom' %}
 Requires=opennsl-modules.service
-{% elif sonic_asic_platform == 'nephos' %}
-Requires=nps-modules-4.9.0-9-2-amd64.service
-{% endif %}
-After=database.service updategraph.service
-After=interfaces-config.service
-{% if sonic_asic_platform == 'broadcom' %}
 After=opennsl-modules.service
-{% elif sonic_asic_platform == 'nephos' %}
+{%- endif %}
+{%- if sonic_asic_platform == 'nephos' %}
```

Comment:
small suggestion: lines 10 and 14 can be optimized by using elif

Reply:
@andriymoroz-mlnx done